specification/subscription/resource-manager/Microsoft.Subscription/stable/2021-10-01/subscriptions.json

```diff
@@ -781,6 +781,10 @@
           "description": "The Management Group Id.",
           "type": "string"
         },
+        "createdTime": {
```

Comment:
>createdTime [](http://example.com/codeflow?start=9&length=11)

Please ensure you have a task to implement the common systemData contract in the next API version:  [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources)